bundle/Resources/js/app.js

```diff
@@ -0,0 +1,35 @@
+
+import './basePath.js';
```

Comment:
If there's no technical reason why this is separated to a different file, I'd just include it here.

Reply:
As ckeditor is now used as a separate build, this import is not needed so file is deleted in https://github.com/netgen/ibexa-fieldtype-html-text/pull/3/commits/c93c56cfb6716a82db7e6f5b25d923edcc68027e